Travis fails with latest master

We've got a failing master with [the latest run][m]:

```
=================================== FAILURES ===================================
____________________________ test_run_reproman_help ____________________________

    @assert_cwd_unchanged
    def test_run_reproman_help():
        out, err = check_run_and_get_output("reproman --help")
        ok_startswith(out, "Usage: ")
>       eq_(err, "")

reproman/tests/test_installed.py:34: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

a = '/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/cryptography/hazmat/bindings/openssl/binding.py:163: ..., please upgrade. A future version of cryptography will drop support for it.\n  utils.C
ryptographyDeprecationWarning\n'
b = '', msg = None

    def assert_equal(a, b, msg=None):
>       assert a == b, msg or "{!r} != {!r}".format(a, b)
E       AssertionError: '/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/cryptography/hazmat/bindings/openssl/binding.py:163: CryptographyDeprecationWarning: OpenSSL version 1.0.1 is no longer sup
ported by the OpenSSL project, please upgrade. A future version of cryptography will drop support for it.\n  utils.CryptographyDeprecationWarning\n' != ''

reproman/tests/utils.py:46: AssertionError
```

That failure's new since the [last PR run][p].

[m]: https://travis-ci.org/ReproNim/reproman/builds/499326698
[p]: https://travis-ci.org/ReproNim/reproman/builds/498981194

--- a/reproman/tests/test_installed.py
+++ b/reproman/tests/test_installed.py
@@ -9,6 +9,7 @@
 """
 
 from mock import patch
+import pytest
 from .utils import ok_startswith, eq_, \
     assert_cwd_unchanged
 
@@ -31,4 +32,8 @@ def check_run_and_get_output(cmd):
 def test_run_reproman_help():
     out, err = check_run_and_get_output("reproman --help")
     ok_startswith(out, "Usage: ")
+    if "OpenSSL version 1.0.1 is no longer supported" in err:
+        # Running the test on Travis currently has a warning.
+        # https://travis-ci.org/ReproNim/reproman/jobs/499326699#L2162
+        pytest.xfail("err not empty due to OpenSSL warning")
     eq_(err, "")

